src/adapt/csscasc.js

```diff
@@ -1337,6 +1336,98 @@ adapt.csscasc.CheckConditionAction.prototype.getPriority = function() {
     return 5;
 };
 
+/**
+ * @constructor
+ * @extends {adapt.csscasc.ChainedAction}
+ */
+adapt.csscasc.NegateHelperAction = function() {
```

Comment:
Since we may be able to use this action elsewhere (e.g. [`:matches()`](https://drafts.csswg.org/selectors-4/#matches) or [`:has()`](https://drafts.csswg.org/selectors-4/#relational)), a name which represents its functionality (`CheckAppliedAction`?) would be preferable.